applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/validation/BegrepValidator.kt

```diff
@@ -0,0 +1,26 @@
+package no.begrepskatalog.conceptregistration.validation
+
+import no.begrepskatalog.generated.model.*
+import org.hibernate.validator.internal.util.ModUtil
+
+fun isValidBegrep(begrep: Begrep): Boolean = when {
+    begrep.status == null -> false
+    begrep.status == Status.UTKAST -> false
+    begrep.anbefaltTerm.isNullOrBlank() -> false
+    begrep.definisjon.isNullOrBlank() -> false
+    begrep.ansvarligVirksomhet == null -> false
+    !begrep.ansvarligVirksomhet.isValid() -> false
+    else -> true
+}
+
+private fun Virksomhet.isValid(): Boolean = when {
+    id.isNullOrBlank() -> false
+    id.length != 9 -> false
+    !isValidOrganisationNumber(id) -> false
+    else -> true
+}
+
+private fun isValidOrganisationNumber(organisationNumber: String): Boolean {
+    val digits: List<Int> = organisationNumber.dropLast(1).split("").filter { !it.isBlank() }.map { it.toInt() }

```

Comment:
Vil si det blir enklere å lese om du har hver av de forskjellige funksjonene på en ny linje

val digits: List<Int> = organisationNumber
    .dropLast(1)
    .split("")
    .filter { !it.isBlank() }
    .map { it.toInt() }